SignalsIntegrationTests/SignalsIntegrationTests/GetDataArgumentsTests.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+using Domain;
+using Dto.Conversions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SignalsIntegrationTests.Infrastructure;
+
+namespace SignalsIntegrationTests
+{
+    [TestClass]
+    public class GetDataArgumentsTests : TestsBase
+    {
+        private DateTime FromTimestamp { get; } = new DateTime(2029, 1, 1, 0, 0, 0, 0);
+        private DateTime ToTimestamp { get; } = new DateTime(2029, 1, 8, 0, 0, 0, 0);
```

Comment:
Perhaps we should add one test case with `FromTimestamp` > `ToTimestamp` (I'm guessing that the repository would yield empty result, and that seems an acceptable convention)?